app/webpacker/javascript/cookie_preferences.js

```diff
@@ -41,6 +41,7 @@ export default class CookiePreferences {
     const serialized = JSON.stringify(categories);
     Cookies.set(CookiePreferences.cookieName, serialized, {
       expires: CookiePreferences.cookieLifetimeInDays,
+      sameSite: 'Lax',
```

Comment:
Can we update the tests to check for this? and is there a reason we're going `Lax` instead of `Secure`? I don't think we support http requests so can't see why we wouldn't be able to use `Secure`?

Reply:
We can definitely use SameSite 'None' with Secure, but I couldn't see why we would need it.

Reply:
Ah reading into it more I think a `Lax` is probably safer for us and `None` with `Secure` may break some of our tracking pixels... it also looks like we don't test the cookie attributes currently either so I'll leave it up to you if you want to add that in or not now.

Reply:
I believe we don't test the cookie attributes because they are not available to javascript. For example, `Cookies.get('_foo')` will return only the cookie values.

Reply:
Ah yes you're right - I had always assumed the attributes were part of the cookie that was available. I guess we could spy on `Cookies.set` but its probably more hassle than its worth...